--- origsrc/goocanvas-1.90.2/configure.in	2010-11-03 05:55:59.000000000 -0500
+++ src/goocanvas-1.90.2/configure.in	2010-11-05 02:17:57.675045600 -0500
@@ -31,7 +31,7 @@ PKG_CHECK_MODULES(PACKAGE, [$pkg_modules
 AC_SUBST(PACKAGE_CFLAGS)
 AC_SUBST(PACKAGE_LIBS)
 
-GETTEXT_PACKAGE=goocanvas
+GETTEXT_PACKAGE=goocanvas-2.0
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package.])
 
--- origsrc/goocanvas-1.90.2/docs/Makefile.am	2010-06-30 15:31:13.000000000 -0500
+++ src/goocanvas-1.90.2/docs/Makefile.am	2010-11-05 02:29:53.854113300 -0500
@@ -10,6 +10,7 @@ AUTOMAKE_OPTIONS = 1.6
 
 # The name of the module, e.g. 'glib'.
 DOC_MODULE=goocanvas
+DOC_MODULE_VERSION=2.0
 
 # The top-level SGML file. You can change this if you want to.
 DOC_MAIN_SGML_FILE=$(DOC_MODULE)-docs.sgml
