--- origsrc/goocanvas-2.0.1/src/goocanvasitem.c	2011-10-09 13:03:37.000000000 -0500
+++ src/goocanvas-2.0.1/src/goocanvasitem.c	2013-08-05 21:01:57.800908000 -0500
@@ -397,7 +397,7 @@ goo_canvas_item_base_init (gpointer g_if
       /**
        * GooCanvasItem::child-notify
        * @item: the item that received the signal.
-       * @pspec: (type GLib.ParamSpec): the #GParamSpec of the changed child property.
+       * @pspec: (type GObject.ParamSpec): the #GParamSpec of the changed child property.
        *
        * Emitted for each child property that has changed.
        * The signal's detail holds the property name. 
--- origsrc/goocanvas-2.0.1/src/goocanvasitemmodel.c	2011-10-01 10:07:18.000000000 -0500
+++ src/goocanvas-2.0.1/src/goocanvasitemmodel.c	2013-08-05 21:02:30.119756500 -0500
@@ -186,7 +186,7 @@ goo_canvas_item_model_base_init (gpointe
       /**
        * GooCanvasItemModel::child-notify
        * @item: the item model that received the signal.
-       * @pspec: (type GLib.ParamSpec): the #GParamSpec of the changed child property.
+       * @pspec: (type GObject.ParamSpec): the #GParamSpec of the changed child property.
        *
        * Emitted for each child property that has changed.
        * The signal's detail holds the property name. 
@@ -1126,7 +1126,7 @@ goo_canvas_item_model_class_install_chil
  *
  * It finds a child property of a canvas item class by name.
  *
- * Returns: (type GLib.ParamSpec) (transfer none): The #GParamSpec of the child
+ * Returns: (type GObject.ParamSpec) (transfer none): The #GParamSpec of the child
  *  property or %NULL if @class has no child property with that name.
  */
 GParamSpec*
@@ -1152,7 +1152,7 @@ goo_canvas_item_model_class_find_child_p
  *
  * It returns all child properties of a canvas item class.
  *
- * Returns: (array length=n_properties) (element-type GLib.ParamSpec) (transfer container):
+ * Returns: (array length=n_properties) (element-type GObject.ParamSpec) (transfer container):
  *  a newly allocated array of #GParamSpec*. The array must be freed with g_free(). 
  */
 GParamSpec**
