--- origsrc/goocanvas-2.0.1/src/Makefile.am	2011-10-01 10:07:18.000000000 -0500
+++ src/goocanvas-2.0.1/src/Makefile.am	2011-11-23 21:37:41.546018300 -0600
@@ -157,7 +157,7 @@ introspection_sources = $(libgoocanvas_2
 GooCanvas-2.0.gir: libgoocanvas-2.0.la
 GooCanvas_2_0_gir_SCANNERFLAGS = --identifier-prefix=Goo --symbol-prefix=goo --warn-all
 GooCanvas_2_0_gir_INCLUDES = Gtk-3.0
-GooCanvas_2_0_gir_CFLAGS = $(INCLUDES)
+GooCanvas_2_0_gir_CFLAGS = $(INCLUDES) -I$(srcdir)
 GooCanvas_2_0_gir_LIBS = goocanvas-2.0
 GooCanvas_2_0_gir_FILES = $(introspection_sources)
 INTROSPECTION_GIRS += GooCanvas-2.0.gir
